app/renderer/radon/index.ts

```diff
@@ -0,0 +1,829 @@
+import { objectValues } from "../utils/objectValues"
+
+export enum RadonArgumentType {
```

Comment:
This is identical to the `RadonType` definition

Reply:
They are not identical. We can use enums to avoid string repetition in method definition and by the other hand, we can use the type to guarantee a function signature.

Reply:
:ok_hand: